movienager/movies/templates/movie/forms/movie-add.html

```diff
@@ -0,0 +1,5 @@
+{% extends 'user_forms/user-edit.html' %}
```

Comment:
This way accounts and movies apps are tightly coupled. Would be better to have base forms on a separate shared package - e.g. `shared-templates`